packages/launcher/.gitignore

```diff
@@ -1,3 +0,0 @@
-# transpiled JS files
```

Comment:
Did you mean to remove these `.gitignore` files? We don't want to commit the transpiled files, do we?

Reply:
yes, they were ignoring transpiled js files that get emitted into the src folders. I prefer them untracked so `git clean -df` properly removes them. This also was useful in the CI script